mods/other/tsm_chests/init.lua

```diff
@@ -58,10 +59,17 @@ minetest.register_node("tsm_chests:chest", {
 		local inv = meta:get_inventory()
 		inv:set_size("main", 8*4)
 	end,
-	can_dig = function(pos,player)
+	on_destruct = function(pos)
 		local meta = minetest.get_meta(pos);
 		local inv = meta:get_inventory()
-		return inv:is_empty("main")
+		-- iterate over chest contents, drop all
+		for index, stack in pairs(inv:get_list("main")) do
+			if not minetest.is_yes(minetest.settings:get("remove_items")) then
+				local obj = minetest.add_item(pos, stack:take_item(stack:get_count()))
+				_ = obj and obj:set_velocity({x = math.random(-1, 1), y = 5, z = math.random(-1, 1)})
```

Comment:
Is this variable even being used? It should be marked as `local` (see [LuaCheck](https://github.com/MT-CTF/capturetheflag/pull/748/checks?check_run_id=2042105702) checks).
```suggestion
				local _ = obj and obj:set_velocity({x = math.random(-1, 1), y = 5, z = math.random(-1, 1)})
```

Reply:
I thought underscore was a throwaway variable.

Reply:
Anyway, the purpose of line 69 is just to call obj:set_velocity after checking obj is not nil.